doc/news/2021-02-01_0.9.4.md

```diff
@@ -0,0 +1,375 @@
+- Debian Sid: update to clang 11. _(Mihael Pranjić)_
+
+## Infrastructure
+
+### Cirrus
+
+- Upgrade Cirrus Fedora docker image to Fedora 33. _(Mihael Pranjić)_
+- Upgrade to Ruby 3.0 for macOS builds. _(Mihael Pranjić)_
+
+### GitHub Actions
+
+- We added a build job that translates Elektra with GCC on macOS. _(Mihael Pranjić, René Schwaiger)_
+
+### Jenkins
+
+- We refactored shared code between pipelines into a [Jenkins Shared Library](https://github.com/ElektraInitiative/jenkins-library). _(Robert Sowula)_
+- We now use Debian Sid to build the documentation instead of Debian Stretch. The Doxygen version in Debian stretch [contains a bug](https://github.com/doxygen/doxygen/issues/6456) that causes the generation of the PDF documentation to fail. _(René Schwaiger)_
+- Use Fedora 33 and 32, drop Fedora 31 use in Jenkins. _(Mihael Pranjić)_
+- The Main and Release Pipeline now creates packages for Debian Buster, Ubuntu Bionic, Ubuntu Focal and Fedora-33. These packages are also installed and automatically tested before they are published. To install these packages, please refer to our [Install documentation](../INSTALL.md). _(Robert Sowula)_
+- We updated our Release Pipeline to push changes directly to our git repositories. _(Robert Sowula)_
+
+### Travis
+
+- Move macOS GCC 10 build job to Github Actions. _(Mihael Pranjić)_
+
+## Website
+
+The website is generated from the repository, so all information about
+plugins, bindings and tools are always up to date.
+
+## Outlook
+
+We are currently working on following topics:
+
+- Elektrify KDE _(Dardan Haxhimustafa)_, _(Felix Resch)_ and _(Mihael Pranjić)_
+- 1.0 API _(Stefan Hanreich)_ and _(Klemens Böswirth)_
+- Improve Java Development Experience _(Michael Tucek)_
+- Elektrify GNOME _(Mihael Pranjić)_
+- Continious Releases _(Robert Sowula)_
+- KDB access using FUSE _(Alexander Firbas)_
+- Default TOML plugin _(Jakob Fischer)_
+- Improve Plugin Framework _(Vid Leskovar)_
+- Improve 3-way merge _(Dominic Jäger)_
+- Shell completion _(Ulrike Schäfer)_
+- Ansible bindings _(Thomas Waser)_
+
+## Statistics
+
+<<`scripts/git-release-stats 0.9.VER-1 0.9.<<VERSION>>`>>
+
+## Join the Initiative!
+
+We welcome new contributors!
+Read [here](https://www.libelektra.org/devgettingstarted/ideas) about how to get started.
+
+As first step, you could give us feedback about these release notes.
+Contact us via our [issue tracker](https://issues.libelektra.org).
+
+## Get the Release!
+
+You can download the release from [here](https://www.libelektra.org/ftp/elektra/releases/elektra-0.9.<<VERSION>>.tar.gz)
+or [GitHub](https://github.com/ElektraInitiative/ftp/blob/master/releases/elektra-0.9.<<VERSION>>.tar.gz?raw=true)
+
+The [hashsums are:](https://github.com/ElektraInitiative/ftp/blob/master/releases/elektra-0.9.<<VERSION>>.tar.gz.hashsum?raw=true)
+
+<<`scripts/generate-hashsums elektra-0.9.<<VERSION>>.tar.gz`>>
+
+The release tarball is also available signed by Mihael Pranjić using GnuPG from
```

Comment:
The release tarball actually already gets signed inside the release pipeline and the signed version then also gets published, or do you actually want to sign it manually with your GPG key?

Reply:
It would be great to use that automatically signed one. The release notes were not updated about which GPG key it is, I only fixed that it is not me :wink:

Reply:
No I misunderstood that. Let's use the bot! Do you have the key fingerprint at hand to update it?

Btw:
One of the steps before triggering the release pipeline is to move the news file to the final place. The link checker will inevitably fail as the tarball etc is not available. This again will block the release pipeline from building the tarball (I am guessing).

Should we simply postpone moving the news file until after the pipeline is done?

Reply:
> Do you have the key fingerprint at hand to update it?

Unfortunatly no, to access it we will probably need to print it inside a Jenkins job, since accessing credentials from Jenkins is apparently not possible. But I have a gpg signed file from the release pipeline from which we can get the key that was used to sign the file, if this is enough?

```sh
$ gpg --verify /Users/robertsowula/Downloads/0.9.3\ 21/elektra-0.9.3.tar.gz.gpg
gpg: Signatur vom Sa 16 Jan 16:39:50 2021 CET
gpg:                mittels RSA-Schlüssel 12CC44541E1B8AD9B66AFAD55262E7353324914A
gpg: Signatur kann nicht geprüft werden: No public key
```

> Should we simply postpone moving the news file until after the pipeline is done?

Thank you for pointing this out, I didn't think of this. Yes I think we have no way around this, we also did it that way in the last release. Is it actually possible to exclude a file from link-checker with a comment (or similar)? Maybe this would be more elegant since we would then only need to remove the comment from this file after the pipeline is done.

Reply:
> key that was used to sign the file, if this is enough?

Yep, thank you!

> Is it actually possible to exclude a file from link-checker

The simplest way is to add a link to `tests/linkchecker.whitelist` in general. The `doc/news/_preparation_next_release.md` is already a special case where links containing `<<VERSION>>` are excluded from the check. It seems like the simplest to just keep this name until after the release, which avoids all those problems.